notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<ul><li>Give a brief explanation how to implement a Pytorch model in words before this cell.</li><li>In the <code>forward</code> method, please directly return <code>self.linear_relu_stack(x)</code> rather than defining the unnecessary variable <code>y</code></li><li>Explain why we need <code>model = NeuralNetworkModel().to(device)</code> and what it does </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Added the explanation. Thanks!</p>